source/testing.rst

```diff
@@ -0,0 +1,57 @@
+.. _testing:
+
+Testing
+=======
+
+DHIL Symfony applications use `PHPUnit Testing Framework`_ to run automated tests on the code components. PHPUnit manual is `here <https://phpunit.de/manual/6.3/en/index.html>`_. A comprehensive tutorial can be found `here <https://jtreminio.com/2013/03/>`_.
+
+.. note:: PHPUnit is enabled after installing Composer and CEWW specific composer dependencies. It should be located at ceww/vendor/bin/phpunit. If, for some reason, this is not the case, simply follow the installation guide `here <https://phpunit.de/manual/current/en/installation.html>`_. Make sure you use the right version (supporting PHP 5.5.9 or later).
+
+1. Navigate into the ceww main directory. Check to see if the phpunit is active:
+
+.. code-block:: bash
+
+  vendor/bin/phpunit --version
+
+2. In order to run a particular test class, type: 
+
+.. code-block:: bash
+
+  vendor/bin/phpunit <path_of_the_file>
+
+Alternatively, you can run the entire test suite by omitting the file path:
+
+.. code-block:: bash
+
+  vendor/bin/phpunit 
+
+.. note:: running the entire test suite can take a lot of time. You can fasten this process by replacing the "phpunit.xml.dist" file with the "phpunit.xml" (simply remove the extension).
+
```

Comment:
You also need to change some of the settings in the phpunit.xml file to make it go faster. I'll send you the settings that I like to use.

The phpunit.xml.dist file is intended to run the test suite reliably and on minimal hardware. You can speed up the test suite if you allow PHP to use unlimited memory and cache a lot of the code. But then you risk running out of memory on your computer.

Reply:
got it. I've linked the manual for the configuration file... fixed!